src/pages/[platform]/build-a-backend/q-developer/index.mdx

```diff
@@ -0,0 +1,162 @@
+
+### Choosing your authentication method
+
+If you plan to use Amazon Q Developer - Inline completions with VS Code (through AWS Toolkit) or JetBrains (through AWS Toolkit), you will have to authenticate using either AWS Builder ID or IAM Identity Center.
+
+If you plan to use Amazon Q Developer - Inline completions with AWS Cloud9 AWS Lambda, SageMaker Studio, JupyterLab, or AWS Glue Studio, you will have to authenticate using IAM.
+
+For information about authenticating with Amazon Q Developer - Inline completions, see [Authenticating with Amazon Q Developer - Inline completions and AWS Toolkit](https://docs.aws.amazon.com/codewhisperer/latest/userguide/whisper-setup-ide-devs.html#codewhisperer-auth).
+
+
+### Setting up your authentication method
+
+- [Builder ID](https://docs.aws.amazon.com/signin/latest/userguide/sign-in-aws_builder_id.html) (used with AWS Toolkit and VS Code or JetBrains) requires only an email address. To use it, you don't even need an AWS account.
+- [IAM Identity Center](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html) requires setup by your enterprise administrator.
+- [IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) credentials are used within your AWS account to regulate access to, and between, various AWS services.
+
+To access AWS, you must sign up for an [AWS account](https://docs.aws.amazon.com/codewhisperer/latest/userguide/setting-up.html#getting-started-signup)
+
+## Getting started with Q Developer - Inline completions in VS Code and JetBrains
+
+<BlockSwitcher>
+<Block name="VS Code">
+
+1. From the AWS Toolkit for VS Code, in the AWS pane, under **CodeWhisperer**, choose **Sign in to get started**. The **AWS Toolkit: Add Connection to AWS** tab will open.
+1. Select the **Amazon Q + CodeWhisperer** authentication panel.
+1. Select the appropriate [authentication method](https://docs.aws.amazon.com/codewhisperer/latest/userguide/whisper-setup-ide-devs.html#codewhisperer-auth) and log in.
+
+![A screenshot of AWS Toolkit for VS Code that displays options with sign in mechanisms for Amazon Q + CodeWhisperer, AWS Explorer, and Amazon CodeCatalyst](/images/gen2/q-developer/VSCode.png)
+
+</Block>
+<Block name="JetBrains">
+
+1. From the AWS Toolkit for JetBrains, select the **Amazon Q + CodeWhisperer** tab.
+1. Under CodeWhisperer, choose **Sign in to get started**. The **AWS Toolkit: Setup Authentication** modal will open.
+1. Select the appropriate [authentication method](https://docs.aws.amazon.com/codewhisperer/latest/userguide/whisper-setup-ide-devs.html#codewhisperer-auth) and log in.
+
+![A screenshot of AWS Toolkit for JetBrains that displays tabs with sign in options for Amazon Q + CodeWhisperer using IAM Identity Center, AWS Builder ID, and IAM credentials](/images/gen2/q-developer/VSCode.png)
```

Comment:
This screenshot doesn't show Jetbrains, this is showing VSCode.

Reply:
Thanks for catching this, your suggested changes will fix the img for JetBrains.